### ISSUE TITLE
Publish an `arm` snap

### ISSUE THREAD

**Issue by mattkae** [mattkae/miracle-wm#59]
We should move out publishing to the CI, which would be very nice IMO. This shouldn't be hard to do if we had a "release" branch, where pushing to it always triggers a release

**Comment by mattkae** [comment]
Maybe investigate using https://packit.dev/

**Comment by sudopluto** [comment]
can i add a tar of  x86_64 binary(ies?) to the wishlist? that way we can have a miracle-wm-bin package in the aur, so people dont have to compile from source. otherwise probably going to create one based on extracting the binaries from the .deb releases

**Comment by mattkae** [comment]
> can i add a tar of x86_64 binary(ies?) to the wishlist? that way we can have a miracle-wm-bin package in the aur, so people dont have to compile from source. otherwise probably going to create one based on extracting the binaries from the .deb releases

Yeah that seems appropriate! I always mean to get back to this task, but it is trickier than I like
